src/Common/src/Interop/Windows/Interop.BOOL.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal partial class Interop
+{
+    internal enum BOOL : int
+    {
+        FALSE = 0,
+        TRUE = 1,
```

Comment:
In most cases, `BOOL` is exactly either 0 or 1. The same potential problem exists in C++. 

The CoreCLR repo has hundreds places with `fSomeFlag == TRUE`...

The defensive coding against this is always compare against `BOOL.FALSE`, never against `BOOL.TRUE`. I will add a comment about it here.